src/main/java/uk/nhs/digital/nhsconnect/lab/results/sequence/SequenceService.java

```diff
@@ -6,18 +6,12 @@
 
 @Service
 public class SequenceService {
-    private static final String TRANSACTION_KEY_FORMAT = "TN-%s";
     private static final String INTERCHANGE_FORMAT = "SIS-%s-%s";
     private static final String INTERCHANGE_MESSAGE_FORMAT = "SMS-%s-%s";
 
     @Autowired
     private SequenceRepository sequenceRepository;
 
-    public Long generateTransactionNumber(final String generalPractitioner) {
-        validateSender(generalPractitioner);
```

Comment:
`validateSender()` is now unused and can be removed

Reply:
Have removed this unused method in 697ebb8